addons/binding/org.openhab.binding.cm11a/README.md

```diff
@@ -0,0 +1,101 @@
+# Cm11a (X10 controller) Binding
+
+The cm11a is a serial computer interface that allows the computer control attached X10 modules. See the Hardware section below for a further description of the cm11a. This is implemented as an OpenHAB 2 binding.
+
+##Credit
+
+This binding is made possible because of the previous work done by "Anthony Green". His work can be seen at: <https://github.com/engineergreen/openhab/tree/cm11a-binding/bundles/binding/org.openhab.binding.cm11a>
+
+## Supported things
+
+The binding currently supports the following thing types:
+
+* switch
+* dimmer
+
+## Discovery
+
+Discovery is not supported because that kind of information is not available in the cm11a or the X10 specification.
+
+## X10 powerline monitoring
+
+The cm11a has the ability to capture x10 messages on the powerline. This binding captures those messages, decodes them and updates the item state. If the UI has a channel open to the server those changes will be reflected in the UI. Otherwise, a refresh of the browser will show the current state.
+
+##Configuration
+
+This binding is an OpenHAB 2 binding and uses the Bridge / Thing design with the cm11a being the Bridge and the controlled modules being the Things. The following definitions are specified in the .things file.
+
+### Bridge Configuration
+
+The bridge requires the serial port used to connect the cm11a to the computer.
+
+    Bridge cm11a:cm11a:MyCm11a  [ serialPort="COM3" ] or
+    Bridge cm11a:cm11a:MyCm11a  [ serialPort="/dev/ttyUSB0" ]
+
+
+### Thing Configuration
+
+Each attached thing must specify whether it is a dimmer or a switch and the house/unit code (i.e. A1).
+
+    Thing switch SwitchA1 [ HouseUnitCode="A1" ]
+    Thing dimmer DimmerA2 [ HouseUnitCode="A2" ]
+
+### Example
```

Comment:
Instead of doing separate examples for things/items/sitemaps I would prefer a single "Full example" as we have it for all the other bindings. No need to explain the syntax in detail as this should be common to all bindings (and thus explained once at a central place in our docs).